sklearn/feature_selection/from_model.py

```diff
@@ -17,11 +18,21 @@ def _get_feature_importances(estimator):
     importances = getattr(estimator, "feature_importances_", None)
 
     if importances is None and hasattr(estimator, "coef_"):
-        if estimator.coef_.ndim == 1:
-            importances = np.abs(estimator.coef_)
+        if isinstance(estimator.coef_, np.ndarray):
+            if estimator.coef_.ndim == 1:
+                importances = np.abs(estimator.coef_)
+            else:
+                importances = np.sum(np.abs(estimator.coef_), axis=0)
+
+        elif sps.isspmatrix_csr(estimator.coef_):
+            importances = np.sum(np.abs(estimator.coef_.toarray()), axis=0)
```

Comment:
Sorry, been busy the past day or so and haven't had much of a chance to get on GitHub. 

Last night, I changed the line in question to `importances = np.sum(abs(estimator.coef_), axis=0) `, and now I get a ValueError from `transform` in the base class which says "X has a different shape than during fitting.". I'm not sure why that's happening, and I need to look into it more.